Document/CategoryDocument.php

```diff
@@ -85,6 +87,16 @@ public function addExpiredUrl($expiredUrl)
     }
 
     /**
+     * Remove expired url from array.
+     *
+     * @param string $expiredUrl
+     */
+    public function removeExpiredUrl($expiredUrl)
```

Comment:
This should be located in [SeoAwareTrait](https://github.com/ongr-io/RouterBundle/blob/master/Document/SeoAwareTrait.php).

Also, `$urls` and `$expiredUrls` properties are defined. Shouldn't they be included using that SeoAwareTrait?